plugin-server/src/main/ingestion-queues/batch-processing/each-batch-async-handlers.ts

```diff
@@ -87,3 +91,38 @@ export async function eachBatchWebhooksHandlers(
 ): Promise<void> {
     await eachBatch(payload, queue, eachMessageWebhooksHandlers, groupIntoBatches, 'async_handlers_webhooks')
 }
+
+async function runWebhooks(hub: Hub, event: PostIngestionEvent) {
+    const timer = new Date()
+
+    try {
+        hub.statsd?.increment('kafka_queue.event_pipeline.start', { pipeline: 'webhooks' })
+        await processWebhooksStep(hub, event)
+        hub.statsd?.increment('kafka_queue.webhooks.processed')
+        hub.statsd?.increment('kafka_queue.event_pipeline.step', { step: processWebhooksStep.name })
+        hub.statsd?.timing('kafka_queue.event_pipeline.step.timing', timer, { step: processWebhooksStep.name })
+    } catch (error) {
+        hub.statsd?.increment('kafka_queue.event_pipeline.step.error', { step: processWebhooksStep.name })
+
+        if (error instanceof DependencyUnavailableError) {
+            // If this is an error with a dependency that we control, we want to
+            // ensure that the caller knows that the event was not processed,
+            // for a reason that we control and that is transient.
+            status.error('Error processing webhooks', {
+                stack: error.stack,
+                eventUuid: event.eventUuid,
+                teamId: event.teamId,
+                error: error,
+            })
+            throw error
+        }
+
+        status.warn(`⚠️`, 'Error processing webhooks, silently moving on', {
+            stack: error.stack,
+            eventUuid: event.eventUuid,
+            teamId: event.teamId,
+            error: error,
+        })
+        silentFailuresAsyncHandlers.inc()
```

Comment:
should we just throw in these cases? - not part of this PR, but you know if we break webhooks

Reply:
yup it is, but since you touched nearby code .... want to put up a follow-up to throw 🤷‍♀️